### PR TITLE
qa_crowbarsetup: Wait for instance to be up when starting VM for manila

### PR DESCRIPTION
**Pull request by vuntz**
Also stop everything if the boot failed.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3145,10 +3145,12 @@ function oncontroller_manila_generic_driver_setup()
         --name $neutron_net $neutron_net 192.168.180.0/24
     fixed_net_id=`neutron net-show fixed -f value -c id`
     manila_service_net_id=`neutron net-show $neutron_net -f value -c id`
-    nova boot --flavor 100 --image manila-service-image \
+    nova boot --poll --flavor 100 --image manila-service-image \
         --security-groups $sec_group,default \
         --nic net-id=$fixed_net_id \
         --nic net-id=$manila_service_net_id manila-service
+
+    [ $? != 0 ] && complain 43 "nova boot for manila failed"
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: I was wondering about that; is there any case where we'd want to continue? I would guess even though the instance fails to spawn, we should be able to get all the info we need to deploy manila and do some other tests in testsetup, for instance.

**Comment by vmoravec** [comment]
+1 thanks

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: You recognize a non-usable instance when you try to create a manila share (manila-share.log will have some errors).   We are currently not testing anything for Manila so it doesn't matter if the VM boots or not. But failing early here sounds good.

**Comment by toabctl** [comment]
+1

**Pull request merged**